Library/Homebrew/utils/analytics.rb

```diff
@@ -69,12 +77,67 @@ def report(type, metadata = {})
         nil
       end
 
-      def report_event(category, action, label = os_arch_prefix_ci, value = nil)
-        report(:event,
-               ec: category,
-               ea: action,
-               el: label,
-               ev: value)
+      def report_influx(category, metadata = {})
+        return unless ENV["HOMEBREW_ANALYTICS_ENABLE_INFLUX"]
+
+        # Rewrite to better visualize data
+        metadata[:info]["on_request"] = true if metadata[:on_request] == true
+
+        # Move info one level up
```

Comment:
Could this be handled by the caller and/or make the Google function do this to the InfluxDB one is a bit cleaner?

Reply:
I guess I can rework it to use a `kwargs` type parameter